test/prelude.t.sol

```diff
@@ -117,6 +130,7 @@ contract PreludeTest is Test, ArrayTestHelper {
         uint256 bidId;         // if none specified a new bid will be placed
         uint256 validatorSize; // if none specified default to 32 eth
         bool withdrawable;     // give the eigenpod "validatorSize" worth of withdrawable beacon shares
+        bytes pubkey;          // emulate validator from mainnet
```

Comment:
change comment -> if none specified a random pubkey is generated

Reply:
updated this